tests/Functional/ConfigurableLanguageTest.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_multilingual\Functional;
+
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests features implemented for the configurable language.
+ */
+class ConfigurableLanguageTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_multilingual',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    /** @var \Drupal\user\UserInterface $user */
+    $user = $this->createUser([], '', TRUE);
```

Comment:
It is better to avoid using tests admin user.
I suppose use user with this permission should be enough: _administer languages, access administration pages, view the administration theme_.